evol/step.py

```diff
@@ -88,8 +88,13 @@ def apply(self, population) -> Population:
             return population.log(**self.kwargs)
         return population
 
-class RepeatStep(EvolutionStep):
 
+class RepeatStep(EvolutionStep):
     def apply(self, population) -> Population:
         return population.evolve(**self.kwargs)
-    # TODO: make a nice __repr__ that is recusive for repeat step and shows repeats 'n'
```

Comment:
Don't we want to keep this TODO?

Reply:
I've moved this to a github issue instead.